color.go

```diff
@@ -345,6 +345,11 @@ func Cyan(format string, a ...interface{}) { printColor(format, FgCyan, a...) }
 func White(format string, a ...interface{}) { printColor(format, FgWhite, a...) }
 
 func printColor(format string, p Attribute, a ...interface{}) {
+	if len(a) == 0 {
+		a = append(a, format)
+		format = "%s"
+	}
```

Comment:
I assume this fixes the cases for `color.White`, `color.Cyan`, etc.. However there are no tests for these. Can you please also add those ?